build/cmake/lib/CMakeLists.txt

```diff
@@ -22,7 +22,11 @@ include_directories(${LIBRARY_DIR} ${LIBRARY_DIR}/common)
 
 file(GLOB CommonSources ${LIBRARY_DIR}/common/*.c)
 file(GLOB CompressSources ${LIBRARY_DIR}/compress/*.c)
-file(GLOB DecompressSources ${LIBRARY_DIR}/decompress/*.c ${LIBRARY_DIR}/decompress/*.S)
+if (MSVC)
+    file(GLOB DecompressSources ${LIBRARY_DIR}/decompress/*.c)
```

Comment:
You should also add the compiler flag `-DZSTD_DISABLE_ASM` in this case. So that the assembly implementation is explicitly disabled.

Reply:
I hope I addressed your comments.